src/components/Instructions.tsx

```diff
@@ -20,67 +28,115 @@ type Step = {
+        will ask you what is you goal for playing today
+        <br />
+        Please tell us why you are playing the game. This helps us understand
+        your needs and continue improving HeaRT tool for you!❤️
+      </div>
+    ),
+    image: stepFourImg,
+  },
+  5: {
+    title: "Pick the statement",
     description: (
       <div>
         You will be presented with the first pair of cards. <br /> If the
         statements on the cards are applicable to your course, pick one that
         resonates with you the most 💓 <br /> If the statements are not
-        applicable, don't worry, you can always skip to the next pair! ⏭️ You
-        must complete at least 8 pairs to finish the game. <br /> Psst🤫...in
-        the future versions we will add the ability to edit statements, so you
-        can tailor the HeaRT game to your course.
+        applicable, don't worry, you can edit them to be more specific to your
+        course OR you can always skip to the next pair! ⏭️ <br /> You must
+        complete at least 8 pairs to finish the game. <br />{" "}
       </div>
     ),
-    image: stepTwoImg,
+    image: stepFiveImg,
   },
-  3: {
-    title: "Select your course dimension",
+  6: {
+    title: "Choose the position on the dimension bar",
     description: (
       <div>
         When you pick a statement you are presented with a dimension bar. You
         need to select your stance on the dimension: how much does your course
         tend towards one card over the other. <br /> For example, is your course
         more teacher-managed, or is it more student-managed? 🤔 <br />
-        In future versions, you will be able to explain your choice for each
-        dimension in a text note.
+        You can also add a note to explain your choice 📜
       </div>
     ),
-    image: stepOneImg,
+    image: stepSixImg,
   },
-  4: {
-    title: "Preview and edit your chart",
+  7: {
+    title: "Preview your chart",
     description: (
       <div>
-        Once you went through all teaching dimensions, you can preview and edit
-        your answers before saving your chart 💾
+        Once you went through all teaching dimensions, you can preview your
+        answers and go back to any card you would like to edit. <br />
+        If you are happy with results, don't hesitate to click 'Save Chart'! 💾
       </div>
     ),
-    image: stepTwoImg,
+    image: stepSevenImg,
   },
-  5: {
-    title: "Save your chart",
+  8: {
+    title: "Compare charts",
     description: (
       <div>
-        {" "}
-        Save your chart, and compare your answers with your colleagues to see
-        what your perspectives on the course are. Discuss the results, and play
-        the game again as needed! <br />
-        The HeaRT tool is all about sparking the conversation 🔥
+        With HeaRT you can also compare up to three charts to each other in
+        order to see how they align with each other. To do that, select two or
```

Comment:
"With HeaRT, you can also compare up to three charts to see how they align with each other. To do this,"

Reply:
Thanks @mitchfaulcon, awesome job on proofreading! Can I hire you to do the same thing for my P4P report? :D

Reply:
Mine needs to be written first hahaha